Sources/Plasma/PubUtilLib/plNetClientComm/plNetClientComm.h

```diff
@@ -151,9 +151,9 @@ void NetCommActivateMsgDispatchers();
 
 // Return this value from your registered msg handler
 // to stop further dispatching of incoming msg.
-const unsigned kOK_MsgConsumed  = hsOK + 1;
+static constexpr hsError kOK_MsgConsumed = hsOK + 1;
```

Comment:
As far as I can tell, nothing ever returns this value, so this message handling code can probably be simplified a bit as well.